lab/04-get-started-with-helm.md

```diff
@@ -0,0 +1,158 @@
+
+* Deployments, Services, configuration ConfigMaps, Secrets
+
+[Helm](https://helm.sh) is a tool that can be installed on any Kubernetes cluster. Helm provides a new concept called a Chart, which groups an arbitrary collection of resources into a logical unit. Let's install `helm` in our cluster:
+
+## Installing Helm
+
+As we did with the `captainkube` application earlier in the lab, let's configure a Service Account and RBAC bindings so Helm has permission to create, modify, and delete resources in our cluster:
+
+```
+kubectl -n kube-system create sa tiller
+kubectl create clusterrolebinding tiller --clusterrole cluster-admin --serviceaccount=kube-system:tiller
+helm init --service-account tiller
+```
+
+**Output**:
+```
+serviceaccount/tiller created
+clusterrolebinding.rbac.authorization.k8s.io/tiller created
+
+Creating /home/jason/.helm
+Creating /home/jason/.helm/repository
+Creating /home/jason/.helm/repository/cache
+Creating /home/jason/.helm/repository/local
+Creating /home/jason/.helm/plugins
+Creating /home/jason/.helm/starters
+Creating /home/jason/.helm/cache/archive
+Creating /home/jason/.helm/repository/repositories.yamlAdding stable repo with URL: https://kubernetes-charts.storage.googleapis.com
+Adding local repo with URL: http://127.0.0.1:8879/charts
+$HELM_HOME has been configured at /home/jason/.helm.
+Happy Helming!
+```
+
+This launches an in-cluster component called `tiller` which interacts with the Kubernetes APIs. We are ready to install some charts.
+
+## Use a chart
+
+Anyone can create a chart, but to get started, let's use a chart that has been authored by the Helm community.
+
+Let's install WordPress, a popular, open source blog. Let's update the list of available charts and look for WordPress:
+
+```
+helm repo update
+```
+
+**Output:**
+```
+Hang tight while we grab the latest from your chart repositories...
+...Skip local chart repository
+...Successfully got an update from the "stable" chart repository
+Update Complete. ⎈ Happy Helming!⎈
+```
+
+Search for WordPress:
+```
+helm search wordpress
+```
+
+**Output**:
+```
+NAME                    CHART VERSION   APP VERSION     DESCRIPTION
+stable/wordpress        3.0.0           4.9.8           Web publishing platform for building blogs and websites.
+```
+
+Great! Looks like WordPress 4.9.8 is available. Let's install this chart on our cluster:
+
+```
+helm install stable/wordpress
+```
+
+As you can see, WordPress is more involved than our simple set of Phippy microservices. A few things to note:
+
+* Helm has generated a `release` for this application, named `exciting-dragonfly`. This lets us run multiple, independent instances of WordPress on the same cluster
```

Comment:
release here would need clarification; maybe a short sentence and a link, per the conversation in slack?